internal/pkg/log/debug.go

```diff
@@ -0,0 +1,37 @@
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/ioutil"
+)
+
+type Logger = zap.SugaredLogger
+
+type DebugLogger struct {
+	*Logger
+	*ioutil.Writer
+}
+
+// NewDebugLogger returns logs as string by String() method.
+// See all methods of the ioutil.Writer.
+func NewDebugLogger() *DebugLogger {
```

Comment:
Commit `log.DebugLogger`

-------------------

Old
```
NewDebugLogger() (*zap.SugaredLogger, *Writer)
```

New
```
NewDebugLogger() *DebugLogger
```

Teda, po starom sa vratil klasicky `logger` + `Writer`, do ktoreho sa logy zapisuju (a ma metodu `String()`, ktora logy vrati).

Je s tym dost problemov. Niekde tie logy potrebujeme, niekde nie, a potom sa musi namiesto 2. hodnoty pisat `_`.

Okrem toho to vyzera nelogicky, vrati ti to `*zap.SugaredLogger, *Writer` a z toho `Writer` mas tie logy citat.

Oba problemy riesit to, ze sa vrati `DebugLogger`.

V tomto PR je este nutne pisat `logger.Logger` .... lebo v celom kode sa pre `logger` nepouziva interface,
ale struktura `*zap.SugaredLogger`. A teda `DebugLogger` nemoze `in-place` nahradit `*zap.SugaredLogger`.

Ale v dalsom PR to bude uz interface, a bude sa dat pouzit `DebugLogger` namiesto `Logger` (obe budu interface).